src/components/Charts/ChartRatioBedsWithCovidPatients.tsx

```diff
@@ -15,6 +15,15 @@ const getTooltipContent = (valueY: number) => ({
   width: '125px',
 });
 
+const maxColumnValue = (data: Column[]) => {
+  return Math.max.apply(
+    null,
+    data.map(function (column) {
+      return column.y;
+    }),
+  );
+};
```

Comment:
I think we want to ensure that the High section shows up even if the max column value is below the Medium-High threshold (15%), I would recommend to set the cap at a minimum of 20% to make sure that the High segment is visible.

Also, consider using `max` from `lodash` to simplify a bit:

```tsx
import max from 'lodash/max';
// ...

// Maybe a better variable name here?
const MIN_Y = 0.2; // 20%

const maxColumnValue = max(data.map(col => col.y));
const capY = Math.max(maxColumnValue + CAP_PADDING, MIN_Y);  
```

Reply:
Ah, yeah good call 🙏 added in 48c4fc3b3467dee2010f17664bf809f98c912129

Example of a county whose max is < 20%: 
<img width="962" alt="image" src="https://user-images.githubusercontent.com/55333380/164567055-808a816d-d75e-4070-ad8f-c87e9f6271df.png">